testdata/scripts/imports.txt

```diff
@@ -1,4 +1,4 @@
-garble build -tags buildtag
+garble -literals build -tags buildtag
```

Comment:
I would suggest to instead add build tags to `literals.txt`. Otherwise we'll end up with many tests mixing in `-literals`, which will make them more complex.

Reply:
It's not about the build tags its about the `reflect.TypeOf()` calls

Reply:
Oh, right, hm. Duplicating all the `reflect` cases in `literals.txt` would be unfortunate.

How about just adding one extra `garble build` step with `-literals` at the very end, not run with `-short`? That way, the "core" short test doesn't mix in `-literals`, and we test all of this with the vanilla lack of options, but we still have the extra sanity test with `-literals` covered by CI.

Reply:
ok